pos_order_return/README.rst

```diff
@@ -111,6 +111,7 @@ Contributors
 * Sylvain LE GAL <https://twitter.com/legalsylvain>
 * David Vidal <david.vidal@tecnativa.com>
 * Kiril Vangelovski <kiril@lambda-is.com>
+* Druidoo <https://www.druidoo.io>
```

Comment:
This file is going to be replaced with the  contents of `readme/CONTRIBUTORS.md`. That's where you need to add Druidoo, not here